### PR TITLE
[T4A2][F12-A2] Lee Jun Han Bryan

### PR DESCRIPTION
**Pull request by bryanleejh**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/ReadOnlyPerson.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
     Phone getPhone();
     Email getEmail();
     Address getAddress();
-
+
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/ReadOnlyPerson.java:16]: Do note for indentation.

File: src/seedu/addressbook/data/person/Person.java [hunk 2]
```diff
@@ -9,11 +9,14 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
-
+	
+	private static int nextSequenceNumber = 1;
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Person.java:13]: Do note the indentation here.

File: src/seedu/addressbook/data/person/Person.java [hunk 3]
```diff
@@ -53,6 +58,10 @@ public Email getEmail() {
     public Address getAddress() {
         return address;
     }
+    
+    public int getSequenceNumber() {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Person.java:62]: Recall from tutorial on commenting of methods.

**Review by brandonyeoxg — COMMENTED** [review]
@bryanleejh Accepting this PR without a review because another of your PRs for this tutorial was reviewed in detail.

File: src/seedu/addressbook/data/person/Person.java (hunk 3 above)
Comment (bryanleejh) [reply src/seedu/addressbook/data/person/Person.java:62]: All public methods should have headers - sorry forgot :/ Thanks Brandon!

**Pull request closed without merging**